diff --git a/node_modules/rescript-relay/src/RescriptRelay.res b/node_modules/rescript-relay/src/RescriptRelay.res
index 4c3e4e3..90fb10f 100644
--- a/node_modules/rescript-relay/src/RescriptRelay.res
+++ b/node_modules/rescript-relay/src/RescriptRelay.res
@@ -601,6 +601,8 @@ module Environment = {
     missingFieldHandlers: array<MissingFieldHandler.t>,
     @optional
     requiredFieldLogger: RequiredFieldLogger.js,
+    @optional
+    isServer: bool,
   }
 
   @module("relay-runtime") @new
@@ -613,6 +615,7 @@ module Environment = {
     ~treatMissingFieldsAsNull=?,
     ~missingFieldHandlers=?,
     ~requiredFieldLogger=?,
+    ~isServer=?,
     (),
   ) =>
     make(
@@ -626,6 +629,7 @@ module Environment = {
         | None => [nodeInterfaceMissingFieldHandler]
         },
         ~requiredFieldLogger=?requiredFieldLogger->Belt.Option.map(RequiredFieldLogger.toJs),
+        ~isServer?,
         (),
       ),
     )
diff --git a/node_modules/rescript-relay/src/RescriptRelay.resi b/node_modules/rescript-relay/src/RescriptRelay.resi
index 9db7790..4d9f0d7 100644
--- a/node_modules/rescript-relay/src/RescriptRelay.resi
+++ b/node_modules/rescript-relay/src/RescriptRelay.resi
@@ -810,6 +810,7 @@ module Environment: {
     ~treatMissingFieldsAsNull: bool=?,
     ~missingFieldHandlers: array<MissingFieldHandler.t>=?,
     ~requiredFieldLogger: RequiredFieldLogger.t=?,
+    ~isServer: bool=?,
     unit,
   ) => t
 
